objects/src/accounts/mod.rs

```diff
@@ -231,7 +231,7 @@ impl Account {
     // TEST HELPERS
     // --------------------------------------------------------------------------------------------
 
-    #[cfg(test)]
+    #[cfg(feature = "testing")]
```

Comment:
is `vault_mut` even used at all? I haven't seen any usages

Reply:
Oh that's true, I think I removed the only usage of this.